apps/opik-backend/src/main/java/com/comet/opik/domain/stats/StatsMapper.java

```diff
@@ -46,12 +48,19 @@ public static ProjectStats mapProjectStats(Row row, String entityCountLabel) {
                 .add(new CountValueStat(METADATA, row.get("metadata", Long.class)))
                 .add(new AvgValueStat(TAGS, row.get("tags", Double.class)));
 
-        BigDecimal totalEstimatedCost = row.get("total_estimated_cost_avg", BigDecimal.class);
-        if (totalEstimatedCost == null) {
-            totalEstimatedCost = BigDecimal.ZERO;
+        BigDecimal totalEstimatedCostAvg = row.get(TOTAL_ESTIMATED_COST_AVG, BigDecimal.class);
+        if (totalEstimatedCostAvg == null) {
+            totalEstimatedCostAvg = BigDecimal.ZERO;
         }
 
-        stats.add(new AvgValueStat(TOTAL_ESTIMATED_COST, totalEstimatedCost.doubleValue()));
+        stats.add(new AvgValueStat(TOTAL_ESTIMATED_COST, totalEstimatedCostAvg.doubleValue()));
```

Comment:
Sorry, I meant to extract this logic a method:

```suggestion
        BigDecimal statsValue = row.get(statsField, BigDecimal.class);
        if (statsValue == null) {
            statsValue = BigDecimal.ZERO;
        }

        stats.add(new AvgValueStat(statsField, statsValue.doubleValue()));
```